helm-chart/renku/templates/swagger.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "renku.fullname" . }}-swagger
+spec:
+  replicas: {{ .Values.replicaCount }}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "renku.name" . }}-swagger
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "renku.name" . }}-swagger
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: swagger
+          image: swaggerapi/swagger-ui
+          env:
+            - name: BASE_URL
+              value: /swagger
+            - name: URLS
+              value: >
+                [
+                  {"url": "/api/renku/spec.json", "name": "core service"},
+                  {"url": "/jupyterhub/services/notebooks/api/v1/spec", "name": "notebooks service"}
+                ]
+            - name: OAUTH2_REDIRECT_URL
+              value: https://{{ .Values.global.renku.domain }}/swagger/oauth2-redirect.html
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /swagger
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /swagger
+              port: http
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+      {{- with .Values.nodeSelector }}
```

```suggestion
      {{- with .Values.swagger.nodeSelector }}
```
same for the values further down.

---

I removed these options, they won't really be needed